estate/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
+"estate.access_estate_property","access_estate_property","estate.model_estate_property","base.group_user",1,0,0,0
```

Comment:
I think you should give all access write to the user.
And there should be one empty line at the end of the file.